### PR TITLE
ParseConfig class + initialization tests

### PR DESCRIPTION
**Pull request by ahemphill**

### CODE REVIEW

**Comment by charbull** [comment]
don't forget to assign me as a reviewer otherwise i will not get notified

File: .github/workflows/python_tools.yml [hunk 1]
```diff
@@ -55,14 +55,14 @@ jobs:
     - name: Configuration Scorer Install dependencies
       run: |
         python setup.py install
-      working-directory: ./tools/scorer
+      working-directory: ./tools/scoring
```
Comment (charbull) [inline .github/workflows/python_tools.yml:58]: thx ! :)

File: tools/scoring/tests/parse_config_test.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+                     dict)  # Dict[str, EntityInstance]
+    self.assertEqual(type(self.parse.parsed['solution']),
+                     dict)  # Dict[str, EntityInstance]
+
+    self.assertEqual(type(self.parse.scores), dict)
+
+  @patch('builtins.print')
+  def testAppendTypes(self, mock_print):
+    self.parse.append_types()
+    self.assertEqual(mock_print.call_count, 4)
```
Comment (charbull) [inline tools/scoring/tests/parse_config_test.py:51]: it is more robust here to assert the content rather than just counting.

**Review by charbull — CHANGES_REQUESTED** [review]

File: tools/scoring/tests/parse_config_test.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  def testInitialize(self):
+    self.assertEqual(self.parse.args['ontology'], self.ontology)
+    self.assertEqual(self.parse.args['solution'], self.solution)
+    self.assertEqual(self.parse.args['proposed'], self.proposed)
+    self.assertFalse(self.parse.args['verbose'])
+
+    self.assertEqual(type(self.parse.universe), ConfigUniverse)
+
+    self.assertEqual(type(self.parse.parsed['proposed']),
+                     dict)  # Dict[str, EntityInstance]
```
[the remark this answers is not in the document]
Comment (ahemphill) [reply tools/scoring/tests/parse_config_test.py:42]: tools/scoring/score/parse_config.py:52

**Comment by ahemphill** [comment]
Closing due to SNAFU with fork; will reopen.

File: tools/scoring/score/parse_config.py [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+    self.universe = BuildUniverse(use_simplified_universe=True)
+    self.parsed = {
+        'proposed': validator.Deserialize([proposed])[0],
+        'solution': validator.Deserialize([solution])[0]
+    }
+    self.scores = {}
+
+  def append_types(self):
+    """
+      Appends types or type names to parsed files
```
[the remark this answers is not in the document]
Comment (ahemphill) [reply tools/scoring/score/parse_config.py:59]: No; as of now they are only used for output. I will revisit and refactor later.

File: tools/scoring/tests/parse_config_test.py (hunk 2 above)
Comment (ahemphill) [reply tools/scoring/tests/parse_config_test.py:51]: I will test more robustly when the method is refactored.

**Pull request closed without merging**